tests/src/test/resources/commandrouter/logback-spring.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Copyright (c) 2020 Contributors to the Eclipse Foundation
+   
+    See the NOTICE file(s) distributed with this work for additional
+    information regarding copyright ownership.
+   
+    This program and the accompanying materials are made available under the
+    terms of the Eclipse Public License 2.0 which is available at
+    http://www.eclipse.org/legal/epl-2.0
+   
+    SPDX-License-Identifier: EPL-2.0
+ -->
+
+<!DOCTYPE configuration>
+
+<configuration>
+
+  <!-- 
+    This is the logging configuration that is used by the
+    Hono Command Router Docker image while executing the integration tests.
+
+    Any changes made here will be reflected on the next start
+    of the Hono Command Router Docker image only, i.e. the next time
+
+    mvn -Prun-tests verify
+
+    is invoked.
+   -->
+
+  <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+    <!-- encoders are assigned the type
+         ch.qos.logback.classic.encoder.PatternLayoutEncoder by default -->
+    <encoder>
+      <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+    </encoder>
+  </appender>
+
+  <root level="INFO">
+    <appender-ref ref="STDOUT" />
+  </root>
+
+  <logger name="org.eclipse.hono" level="DEBUG"/>
+  <logger name="org.eclipse.hono.client" level="TRACE"/>
+  <logger name="org.eclipse.hono.adapter.client" level="TRACE"/>
+  <logger name="org.eclipse.hono.connection" level="TRACE"/>
+  <logger name="org.eclipse.hono.commandrouter" level="TRACE"/>
+  <logger name="org.eclipse.hono.service" level="TRACE"/>
+
+<!--  <springProfile name="trace">-->
+<!--    <logger name="org.eclipse.hono.commandrouter" level="TRACE"/>-->
+<!--    <logger name="org.eclipse.hono" level="DEBUG"/>-->
+<!--  </springProfile>-->
+
+<!--  <springProfile name="dev">-->
+<!--    <logger name="org.eclipse.hono" level="DEBUG"/>-->
+<!--  </springProfile>-->
+
+<!--  <springProfile name="prod">-->
+<!--    <logger name="org.eclipse.hono" level="INFO"/>-->
+<!--  </springProfile>-->
+
```

Comment:
this doesn't look right

Reply:
This is only for debugging the integration test errors (I didn't get them locally). It's in the 2nd commit which won't be merged.

Reply:
I now also get the test errors locally (sometimes occurring in CoAP tests). I think it's an issue with the CommandRouter component. I'm looking into it and will see to fix it outside of this PR.

Reply:
I've removed the above log changes. PR with the fix for the intermittent test failures is in #2323.

Reply:
I've rebased now to include the fix here.